tox.ini

```diff
@@ -13,8 +13,9 @@ deps = .[test,ceph,gnocchi]
        python-swiftclient
        tooz[redis]
 passenv = TMPDIR_FOR_XATTR
+allowlist_externals = env
 commands =
-    stestr run {posargs}
+    env PYTHONPATH={envsitepackagesdir}:/usr/lib/python3/dist-packages/ stestr run {posargs}
```

Comment:
Can just set sitepackages=True above instead of adding path here

Reply:
I tried `sitepackages=True`, but it didn't work. tox is running with a python installed with virtualenv as this Travis log shows:
```
$ source ~/virtualenv/python3.8/bin/activate
```
and the virtual environment doesn't include `/usr/lib/python3/dist-packages/`.

Reply:
Sounds reasonable to me. IIRC Debian/Ubuntu installs global modules in a non-standard path to Python (patched), which won't work with `sitepackages=True` anyway.